watches.yaml

```diff
@@ -3,6 +3,8 @@
   version: v1alpha1
   kind: Konk
   chart: helm-charts/konk
+  overrideValues:
+    certManager.namespace: $CERT_MANAGER_NAMESPACE
```

Comment:
This will mean that users can not override the certManager.namespace

Reply:
Yes, the intent is that we set this for them in the operator, since it will be consistent across the cluster.